packages/common/package.json

```diff
@@ -39,8 +40,10 @@
     "react-dom": "^18.3.1"
   },
   "dependencies": {
+    "@powerhousedao/analytics-engine-core": "^0.5.0",
     "@powerhousedao/builder-tools": "workspace:*",
     "@powerhousedao/design-system": "workspace:*",
+    "@powerhousedao/reactor-api": "workspace:*",
```

Comment:
It shouldn't be necessary to add`@powerhousedao/analytics-engine-core` and `@powerhousedao/reactor-api` as explicit dependencies.
`@powerhousedao/reactor-browser/analytics` reexports most of the exports from `@powerhousedao/analytics-engine-core`.

Reply:
fixed in [ae42d36](https://github.com/powerhouse-inc/powerhouse/pull/1607/commits/ae42d3601cddd6819a1413b032433cfd854e0db7)